docs/rotate-git-crypt.md

```diff
@@ -0,0 +1,55 @@
+# Rotating the git-crypt key
+
+We use `git-crypt` for securing secrets committed to `git`. It uses a symmetric encryption key which is then encrypted for users using their GPG keys.
+
+When rotating the `git-crypt` symmetric key, you should follow the steps below:
+
+1. Ensure you have unlocked the repository: `git-crypt unlock`
+
+2. Make a list of the current users:
+  ```
+  ls .git-crypt/keys/default/0/*.gpg | sed -E 's/^.+\/([A-Z0-9]{40}).gpg$/\1/' > git-crypt-users
+  ```
```

Comment:
This is quite hard to read. How about this?

```
ls .git-crypt/keys/default/0/*.gpg \
  | sed 's/.*default.0.//' \
  | sed 's/.gpg//' \
  > git-crypt-users
```

Reply:
As discussed, I've ended up replacing the regular expressions with simpler commands and comments for the reader. It's hopefully easier to read now.